tools/js-test-runner/package.json

```
@@ -18,12 +19,16 @@
 		"@testing-library/react-hooks": "4.0.1",
 		"@testing-library/user-event": "12.8.3",
 		"@wojtekmaj/enzyme-adapter-react-17": "0.6.3",
+		"@wordpress/jest-preset-default": "7.1.3",
+		"babel-jest": "27.4.5",
```

Since we're depending on jest 27.3.1 (which matches all the existing jest deps elsewhere in the monorepo :1st_place_medal:), perhaps we should depend on the same version of babel-jest instead of a later one?
```suggestion
		"babel-jest": "27.3.1",
```